Assets/Scripts/Map/MapPatrollingGen/WaypointConnection.cs

```diff
@@ -179,9 +146,9 @@ private static List<Vertex> MergeIslandsRecursively(
                 return currentIsland; // Base case: no more islands to merge
             }
 
-            List<Vertex>? closestIsland = null;
-            Vertex? closestVertexInCurrent = null;
-            Vertex? closestVertexInNew = null;
+            List<Vertex> closestIsland = null;
+            Vertex closestVertexInCurrent = null;
+            Vertex closestVertexInNew = null;
```

Comment:
Why is the nullability information removed here?
Do you use vscode? It seems to have problems understanding that nullable is enabled for the whole project.

Reply:
Yes, I use vscode, should I put the ? back?